--- ldb-1.5.1/buildtools/wafsamba/wscript~	2018-08-14 15:34:20.000000000 +0000
+++ ldb-1.5.1/buildtools/wafsamba/wscript	2018-09-02 20:45:48.367917432 +0000
@@ -68,7 +68,7 @@
 
     opt.add_option('--with-modulesdir',
                    help=("modules directory [PREFIX/modules]"),
-                   action="store", dest='MODULESDIR', default='${PREFIX}/modules')
+                   action="store", dest='MODULESDIR', default='${PREFIX}/lib64/ldb/modules')
 
     opt.add_option('--with-privatelibdir',
                    help=("private library directory [PREFIX/lib/%s]" % Utils.g_module.APPNAME),
--- ldb-1.5.1/buildtools/wafsamba/wscript~	2018-09-02 20:45:48.000000000 +0000
+++ ldb-1.5.1/buildtools/wafsamba/wscript	2018-09-02 20:47:38.385915901 +0000
@@ -71,7 +71,7 @@
                    action="store", dest='MODULESDIR', default='${PREFIX}/lib64/modules')
 
     opt.add_option('--with-privatelibdir',
-                   help=("private library directory [PREFIX/lib/%s]" % Utils.g_module.APPNAME),
+                   help=("private library directory [PREFIX/lib64/%s]" % Utils.g_module.APPNAME),
                    action="store", dest='PRIVATELIBDIR', default=None)
 
     opt.add_option('--with-libiconv',
@@ -263,7 +263,7 @@
             cc.run = oldmeth
 
     # make the install paths available in environment
-    conf.env.LIBDIR = Options.options.LIBDIR or '${PREFIX}/lib'
+    conf.env.LIBDIR = Options.options.LIBDIR or '${PREFIX}/lib64'
     conf.env.BINDIR = Options.options.BINDIR or '${PREFIX}/bin'
     conf.env.SBINDIR = Options.options.SBINDIR or '${PREFIX}/sbin'
     conf.env.MODULESDIR = Options.options.MODULESDIR
